code/components/citizen-server-impl/src/state/ServerGameState_Scripting.cpp

```diff
@@ -1093,4 +1093,23 @@ static InitFunction initFunction([]()
 
 		return 0;
 	}));
+
+	fx::ScriptEngine::RegisterNativeHandler("SET_ENTITY_DISTANCE_CULLING_RADIUS", makeEntityFunction([](fx::ScriptContext& context, const fx::sync::SyncEntityPtr& entity)
+	{
+		if (context.GetArgumentCount() > 1)
+		{
+			float radius = context.GetArgument<float>(1);
+
+			if (radius > 0.0f)
+			{
+				entity->overrideCullingRadius = radius;
+			}
+			else
+			{
+				entity->overrideCullingRadius = NULL;
```

Comment:
again, NULL is for a pointer (and won't exist on Linux), `0.0f` is more likely and by then will actually not need an extra `if` (unless you're trying to keep the 'negative float' part).